components/terms_of_service/terms_of_service.jsx

```diff
@@ -156,7 +156,10 @@ export default class TermsOfService extends React.PureComponent {
                 <div>
                     <div className='signup-team__container terms-of-service__container'>
                         <div className={termsMarkdownClasses}>
-                            <div className='medium-center'>
+                            <div
+                                className='medium-center'
+                                data-testid='terms-of-service'
```

Comment:
I'd suggest for consistency
```suggestion
                                data-testid='termsOfService'
```